### ISSUE TITLE
Improve Keyboard Navigation and Screen-Reader Accessibility on Uyuni Website

### ISSUE THREAD

**Issue by Rakielle** [uyuni-project/uyuni-project.github.io#100]
# Details

As part of [openSUSE/mentoring#205](https://github.com/openSUSE/mentoring/issues/205), the goal of this issue is to improve keyboard navigation on the Uyuni website and make it screen-reader friendly. 
# Description

The images below visually represent how a screen reader interprets the contents of the website, highlighting the elements that are skipped and not accessible for a screen reader.
This improvement will help users to navigate and interact with the website using only the keyboard. 
It's beneficial for users with no vision or low vision, and those who may have difficulty controlling a pointing device.

<img src="https://github.com/uyuni-project/uyuni-project.github.io/assets/48498778/ea2e9727-8e68-4b0a-8580-3e9c28aaec91" alt="img1"></br>

<img src="https://github.com/uyuni-project/uyuni-project.github.io/assets/48498778/9767f0b1-a69f-489b-9245-311623f24231" alt="img2"></br>

# Proposed Solution
To improve accessibility and address this issue, we can;
1. Use semantic HTML and ensure that the HTML code follows semantic standards.
2. Use ARIA attributes like role, aria-label, aria-labelledby, and aria-describedby where needed.
